Resources/ServerInfo/CMRules.txt

```diff
@@ -0,0 +1,171 @@
+[color=#ff0000]YOU MUST BE AT LEAST 16 YEARS OF AGE TO PLAY CM14. ANY USERS SUSPECTED OF BEING UNDER 16 YEARS OF AGE WILL BE BANNED UNTIL THEY ARE OF AGE.[/color]
+
+[color=#ff0000]DISCONNECTING FROM OR IGNORING/EVADING ADMIN-HELPS WILL RESULT IN AN APPEAL ONLY BAN.[/color]
+This is to prevent bad actors from avoiding issues. If you need to leave urgently, let us know and it will be considered in your appeal.
+
+You can find clarifications of any ambiguity in the in game guidebook.
+
+These rules are neither exactly SS14 or CM13 and should be read in their entirety.
```

Comment:
This isn't in the guidebook yet

```suggestion
This is to prevent bad actors from avoiding issues. If you need to leave urgently, let us know and it will be considered in your appeal.

These rules are neither exactly SS14 or CM13 and should be read in their entirety.
```